service/openapi/src/main/java/org/dotwebstack/framework/service/openapi/mapping/ResponseMapper.java

```diff
@@ -96,12 +97,27 @@ private Object mapObjectDataToResponse(@NonNull ResponseObject responseObject, O
               result.put(child.getIdentifier(), object);
             }
           } else {
-            dataStack.add(0, data);
-            object = mapObject((Map<String, Object>) data, child, dataStack);
-            if (!(Objects.isNull(object))) {
-              result.put(child.getIdentifier(), object);
+            if (data instanceof Map) {
+              dataStack.add(0, data);
+              object = mapObject((Map<String, Object>) data, child, dataStack);
+              if (!(Objects.isNull(object))) {
```

Comment:
Deze?
```suggestion
              if (Objects.nonNull(object)) {
```
Staat ook op meerdere plekken in deze class